build/pmm-client-docker/entrypoint.py

```diff
@@ -1,3 +1,59 @@
 #!/usr/bin/env python
+'''Basic Kubernetes-compatible PMM Client Docker entrypoint.'''
 
-print("PMM CLIENT 2.X DOCKER")
+from __future__ import print_function, unicode_literals
+from distutils.util import strtobool
+from os import environ, execlp
+from subprocess import Popen
+import sys
+
+
+# if PMM_AGENT_SETUP is True run:
+# `pmm-agent setup --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml`
+PMM_AGENT_SETUP = False
+
+PMM_AGENT_CONFIG = '/usr/local/percona/pmm2/config/pmm-agent.yaml'
+
+
+def pmm_agent_setup():
+    '''Setup pmm-agent.'''
+    cmd = ('pmm-agent', 'setup', '--config-file=%s' % (PMM_AGENT_CONFIG,))
```

Comment:
`('pmm-agent', 'setup', '--config-file', PMM_AGENT_CONFIG)` to handle spaces just in case

Reply:
Looks like it works fine with spaces

```
sh-4.2$ sed -n '21p' /entrypoint.py 
PMM_AGENT_CONFIG = '/usr/local/percona/pmm2/config/pmm agent.yaml'
sh-4.2$ python /entrypoint.py 
setup pmm-agent
INFO[2020-01-31T13:30:01.385+00:00] Loading configuration file /usr/local/percona/pmm2/config/pmm agent.yaml.  component=setup
....
/usr/local/percona/pmm2/exporters/rds_exporter  component=setup
Checking local pmm-agent status...
pmm-agent is not running.
Registering pmm-agent on PMM Server...
Registered.
Configuration file /usr/local/percona/pmm2/config/pmm agent.yaml updated.
Please start pmm-agent: `pmm-agent --config-file=/usr/local/percona/pmm2/config/pmm agent.yaml`.
Starting pmm-agent: `pmm-agent --config-file=/usr/local/percona/pmm2/config/pmm agent.yaml
INFO[2020-01-31T13:30:01.632+00:00] Using /usr/local/percona/pmm2/exporters/node_exporter  component=main
INFO[2020-01-31T13:30:01.635+00:00] Using /usr/local/percona/pmm2/exporters/mysqld_exporter  component=main
....
INFO[2020-01-31T13:30:12.062+00:00] Done.                                         component=main
sh-4.2$ ls -ali /usr/local/percona/pmm2/config
total 16
6947372 drwxr-xr-x 1 pmm-agent pmm-agent 4096 Jan 31 13:30 .
6947335 drwxr-xr-x 1 pmm-agent pmm-agent 4096 Jan 30 14:54 ..
 396340 -rw-r----- 1 pmm-agent pmm-agent  744 Jan 31 13:30 pmm agent.yaml
6947373 -rw-r----- 1 pmm-agent pmm-agent    0 Jan 30 14:54 pmm-agent.yaml
sh-4.2$ cat /usr/local/percona/pmm2/config/pmm\ agent.yaml 
# Updated by `pmm-agent setup`.
---
id: /agent_id/3d156687-f8b0-48b8-aca9-81a2a2af101c
listen-port: 7777
server:
  address: pmm-server:443
  username: admin
  password: admin
  insecure-tls: true
paths:
  exporters_base: /usr/local/percona/pmm2/exporters
  node_exporter: /usr/local/percona/pmm2/exporters/node_exporter
  mysqld_exporter: /usr/local/percona/pmm2/exporters/mysqld_exporter
  mongodb_exporter: /usr/local/percona/pmm2/exporters/mongodb_exporter
  postgres_exporter: /usr/local/percona/pmm2/exporters/postgres_exporter
  proxysql_exporter: /usr/local/percona/pmm2/exporters/proxysql_exporter
  rds_exporter: /usr/local/percona/pmm2/exporters/rds_exporter
  tempdir: /tmp
ports:
  min: 42000
  max: 51999
debug: false
trace: false
```